iam/api-client/service_accounts.py

```diff
@@ -73,27 +87,40 @@ def rename_service_account(email, new_display_name):
     """Changes a service account's display name."""
 
     # First, get a service account using List() or Get()
+    credentials = service_account.Credentials.from_service_account_file(
+        filename=os.environ['GOOGLE_APPLICATION_CREDENTIALS'],
+        scopes=['https://www.googleapis.com/auth/cloud-platform'])
+
+    service = googleapiclient.discovery.build(
+        'iam', 'v1', credentials=credentials)
+
     resource = 'projects/-/serviceAccounts/' + email
-    # pylint: disable=no-member
-    service_account = service.projects().serviceAccounts().get(
+
+    my_service_account = service.projects().serviceAccounts().get(
         name=resource).execute()
 
     # Then you can update the display name
-    service_account['displayName'] = new_display_name
-    service_account = service.projects().serviceAccounts().update(
+    my_service_account['displayName'] = new_display_name
+    my_service_account = service.projects().serviceAccounts().update(
         name=resource, body=service_account).execute()
```

Comment:
This line is throwing an error in the test suite I ran manually and I believe it is because you need to rename "body=service_account" to "body=my_service_account". It's alarming that this test didn't run in CI, it must be disabled for some reason.